src/prognostic_variables.jl

```diff
@@ -96,3 +96,85 @@ function Base.zeros(::Type{PrognosticVariables{NF}},
     pres = zeros(SurfaceLeapfrog{NF},_lmax,_mmax)                     # 2 leapfrog time steps for pres
     return PrognosticVariables(layers,pres,lmax,mmax,N_LEAPFROG,nlev)
 end
+
+# SET_VAR FUNCTIONS TO ASSIGN NEW VALUES TO PrognosticVariables
+
+function set_var!(progn::PrognosticVariables{NF}, 
+                  varname::Symbol, 
+                  var::Vector{<:LowerTriangularMatrix},
+                  lf::Integer=1) where NF
```

Comment:
If these lines are changed to
```julia
                 var::LowerTriangularMatrix...;
                 lf::Integer=1) where NF
```
then both `L` and `(L1,L2)` could be used as input? (as ... just unravels the tuple?) Is that helpful or would that just complicate things further down?